### ISSUE TITLE
double-with characters mess up display

### ISSUE THREAD

**Issue by eMBee** [lxsang/PTerm#10]
there is an issue with the display of double-with characters (like chinese) that i don't know how what to make of. the characters show up as boxes, which is fine (i can't read then anyways :-) but they have the wrong size, which messes up the display of the characters behind.

i'll try to make as screenshot...

btw: pharo can display chinese characters. though i don't know how if that also works with monospace fonts:
http://forum.world.st/Chinese-Character-Data-td4843635.html

**Comment by lxsang** [comment]
Yeah, this is a known issue that i dont figure out how to fix it yet, (it seams something related to Pharo). Event if the character is displayed correctly with Unifont, the tab is not aligned correctly
![](https://github.com/lxsang/online_stuffs/raw/master/dword-test.png)
